packages/vulcan-lib/lib/client/serviceWorker.js

```diff
@@ -0,0 +1,8 @@
+export const createServiceWorker = () => {
```

Comment:
I would prefer calling it `registerServiceWorker`

Reply:
Also, this needs to be opt-in via a setting. I don't want a service worker shoved in my app without explicitely consenting to it;